.github/actions/integration-test-setup/action.yml

```diff
@@ -9,14 +9,14 @@ inputs:
   jdk-version:
     description: JDK version
     required: false
-    default: 17
+    default: "17"
```

Comment:
Does this need to be quoted? I'm curious why if so.

Reply:
The JSON schema for GitHub actions defines this as a string, and YAML will identify it as a number if it is not quoted. The same for boolean values.